app/database_etl/location_utils/country_iso2.json

```diff
@@ -1,59 +1,241 @@
 {
-    "United States of America":"US",
-    "Russian Federation":"RU",
-    "France":"FR",
-    "Brazil":"BR",
-    "Japan":"JP",
-    "Italy":"IT",
-    "Denmark":"DK",
-    "China":"CN",
-    "Egypt":"EG",
-    "United Kingdom":"GB",
-    "Switzerland":"CH",
-    "Germany":"DE",
-    "India":"IN",
-    "Spain":"ES",
-    "Greece":"GR",
-    "Iran":"IR",
-    "Republic of Korea":"KP",
-    "Hungary":"HU",
-    "Sweden":"SE",
-    "Portugal":"PT",
-    "Iceland":"IS",
-    "Canada":"CA",
-    "Netherlands":"NL",
-    "Kenya":"KE",
-    "Austria":"AT",
-    "Norway":"NO",
-    "Colombia":"CO",
-    "Saudi Arabia":"SA",
-    "Belgium":"BE",
-    "Israel":"IL",
-    "Georgia":"GE",
-    "Togo":"TG",
-    "Ethiopia":"ET",
-    "Pakistan":"PK",
-    "Malaysia":"MY",
-    "Turkey":"TR",
-    "Libya":"LY",
-    "Ireland":"IE",
-    "Chile":"CL",
-    "Croatia":"HR",
-    "Ecuador":"EC",
-    "State of Palestine":"PS",
-    "Mexico":"MX",
-    "Thailand":"TH",
-    "Malawi":"MW",
-    "Peru":"PE",
-    "Nigeria":"NG",
-    "Czechia":"None",
-    "Singapore":"SG",
-    "Bulgaria":"BG",
-    "Luxembourg":"LU",
-    "Jordan":"JO",
-    "Australia":"AU",
-    "Argentina":"AR",
-    "Armenia":"AM",
-    "Romania":"RO",
-    "Republic of Congo": "CG"
+  "Afghanistan": "AF",
+  "Albania": "AL",
+  "Algeria": "DZ",
+  "American Samoa": "AS",
+  "Andorra": "AD",
+  "Angola": "AO",
+  "Anguilla": "AI",
+  "Antigua and Barbuda": "AG",
+  "Argentina": "AR",
+  "Armenia": "AM",
+  "Aruba": "AW",
+  "Australia": "AU",
+  "Austria": "AT",
+  "Azerbaijan": "AZ",
+  "Bahamas": "BS",
+  "Bahrain": "BH",
+  "Bangladesh": "BD",
+  "Barbados": "BB",
+  "Belarus": "BY",
+  "Belgium": "BE",
+  "Belize": "BZ",
+  "Benin": "BJ",
+  "Bermuda": "BM",
+  "Bhutan": "BT",
+  "Bolivia (Plurinational State of)": "BO",
+  "Bonaire, Sint Eustatius and Saba": "BQ",
+  "Bosnia and Herzegovina": "BA",
+  "Botswana": "BW",
+  "Brazil": "BR",
+  "British Virgin Islands": "VG",
+  "Brunei Darussalam": "BN",
+  "Bulgaria": "BG",
+  "Burkina Faso": "BF",
+  "Burundi": "BI",
+  "Cabo Verde": "CV",
+  "Cambodia": "KH",
+  "Cameroon": "CM",
+  "Canada": "CA",
+  "Cayman Islands": "KY",
+  "Central African Republic": "CF",
+  "Chad": "TD",
+  "Chile": "CL",
+  "China": "CN",
+  "Christmas Island": "CX",
+  "Cocos (Keeling) Islands": "CC",
+  "Colombia": "CO",
+  "Comoros": "KM",
+  "Congo": "CG",
+  "Cook Islands": "CK",
+  "Costa Rica": "CR",
+  "Croatia": "HR",
+  "Cuba": "CU",
+  "Cura\u00e7ao": "CW",
+  "Cyprus": "CY",
+  "Czechia": "CZ",
+  "C\u00f4te d'Ivoire": "CI",
```

Comment:
Why are accents represented differently in this json file compared to the country_iso3 json file? Have we tested that iso codes for these countries are being looked up properly?

Reply:
In Python, they equate to the same thing (the encoded string) so there is no functional difference. However, I can change the country name in this file to match the other one.
![image](https://user-images.githubusercontent.com/38389364/108798164-20575480-755b-11eb-8ce3-d73be4223e96.png)

Reply:
Country name has been changed.